examples/example_feature_types.py

```diff
@@ -28,35 +28,41 @@
     raise
 
 
-def main():
-    # Load adult dataset from openml.org, see https://www.openml.org/t/2117
-    openml.config.apikey = '610344db6388d9ba34f6db45a3cf71de'
+############################################################################
+# Data Loading
+# ======================================
+# Load adult dataset from openml.org, see https://www.openml.org/t/2117
+openml.config.apikey = '610344db6388d9ba34f6db45a3cf71de'
```

Comment:
Actually, this is no longer necessary and we can use sklearn's fetch_openml function for this https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_openml.html

Could you please replace the data loading with the sklearn function?